app/containers/EmojiPicker/EmojiCategory.js

```diff
@@ -42,10 +44,56 @@ export default class EmojiCategory extends React.Component {
 
 		this.size = Math.min(width || widthWidth, windowHeight) / (emojisPerRow || EMOJIS_PER_ROW);
 		this.emojis = props.emojis;
+		this.state = {
+			isModalVisible: false,
+			modalX: 0,
+			modalY: 0,
+			modalData: []
+		};
 	}
 
 	shouldComponentUpdate() {
-		return false;
+		return true;
```

Comment:
You should test if you really want to update the component instead of updating all the time.